keps/sig-node/2570-memory-qos/README.md

```diff
@@ -635,8 +652,8 @@ Recall that end users cannot usually observe component logs or access metrics.
 - [ ] API .status
   - Condition name: 
   - Other field: 
-- [ ] Other (treat as last resort)
-  - Details:
+- [X] Other (treat as last resort)
```

Comment:
should we send an event on breaches? How frequent do we think they might be? Should the user know that they are pushing the limit?

Reply:
Added details about memory.events in this section. We will support it in Beta instead of GA.